opentracing-jfr-tracer/src/main/java11/io/opentracing/contrib/jfrtracer/impl/jfr/JfrScopeEmitterImpl.java

```diff
@@ -65,8 +67,8 @@ public void start(String parentId, String operationName) {
 		if (currentEvent.isEnabled()) {
 			currentEvent.operationName = operationName;
 			currentEvent.parentId = parentId;
-			currentEvent.traceId = span.context().toTraceId();
-			currentEvent.spanId = span.context().toSpanId();
+			currentEvent.traceId = SpanContextUtil.getTraceIdBySpanContext(span.context());
```

Comment:
Less useful/hardcoded variant of the ContextExtractors (which were pluggable). We just got rid of the implementation dependencies - do not reintroduce them.